### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by olia-trofymchuk — APPROVED** [review]
Everything looks good to me :)

**Comment by HStakhiv** [comment]
image
image
image
image
image
image

File: templates/taxi/car_detail.html [hunk 1]
```diff
@@ -9,4 +9,6 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
```
Comment (Nattalli) [inline templates/taxi/car_detail.html:11]: I don't like this page. Make it more beautiful. Make smaller the word `Drivers`, maybe `h3`or `h4` tag would be better. Make text about Manufacturer bigger

**Review by Nattalli — CHANGES_REQUESTED** [review]
Everything else is cool!

**Comment by HStakhiv** [comment]
Is it better now?
image

**Review by Nattalli — APPROVED** [review]
Much better now!

**Pull request closed without merging**